frontend/src/Components/Indicatorbars/PlantDetailBar.md

```diff
@@ -0,0 +1,9 @@
+```js
+<PlantDetailBar waterDays={3} fertilizerWeeks={2} />
+```
+
+PlantDetailBar: low
```

Comment:
You could also have a headline for the upper part in order to have it consitent.